system-test/spanner.ts

```diff
@@ -44,6 +44,24 @@ const spanner = new Spanner({
   projectId: process.env.GCLOUD_PROJECT,
   apiEndpoint: process.env.API_ENDPOINT,
 });
+const GAX_OPTIONS: CallOptions = {
```

Comment:
This looks great. Is it possible to add the same logic to the samples tests? The quota issue can happen in both types of tests.